packages/scss/src/components/inputs/radio-buttons/_radio-buttons.scss

```diff
@@ -41,45 +43,51 @@
 // PALETTES
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 @mixin radioLabelColoring($palette) {
-	color: _color($palette, "800");
-	box-shadow: inset 0 0 0 1px _color($palette, "200");
+	color: _color($palette, 700);
+	box-shadow: inset 0 0 0 1px _color($palette, 200);
 
 	&:hover {
-		background-color: _color($palette, "50");
-		box-shadow: inset 0 0 0 1px _color($palette, "200");
+		background-color: _color($palette, 50);
+		box-shadow: 0px 1px 2px rgba(19, 29, 53, 0.06), 0px 2px 8px rgba(19, 29, 53, 0.04), inset 0 0 0 1px _color($palette, 200);
```

Comment:
I'm managing both the elevation and border in this `box-shadow`. Should we move the borders to the `border` property instead?

Reply:
we use the same things on button because of double border. I don't think we can do this by another way.